source/common/http/header_utility.h

```diff
@@ -33,17 +33,30 @@ class HeaderUtility {
     const bool invert_match_;
   };
 
+  // A MatchOption specifies how to match the header value.
+  struct MatchOption {
```

Comment:
Do we need this match option? Should we always do path normalization?

Reply:
rbac set this to true by default, for other cases, we should leave it for the caller to decide at this moment.